src/Cpanel/CpanelAPI.php

```diff
@@ -186,7 +186,7 @@ private function uapi($module, $function, $parameters, $value)
      *
      * @return bool
      */
-    public function uapi_response_ok($cpanel_uapi_response)
+    public function uapi_response_ok($cpanel_uapi_response) // @codingStandardsIgnoreLine
```

Comment:
Did you add these `// @codingStandardsIgnoreLine`?

I guess the reason is `under_score_thing` syntax. It's not worth refactoring.